frontend/src/components/inputs/Selector.vue

```diff
@@ -99,4 +107,5 @@ const optionClasses = `flex flex-row items-center gap-8 cursor-pointer
   flex-[0_0_4.5rem] w-full px-8 rounded-xl border border-teal-light text-mint text-2xl
   hover:border-teal-dark hover:bg-teal-dark
 `;
+const highlightedOptionClasses = `!text-teal-light font-semibold`;
```

Comment:
I think the UI would look more intuitive if we make the background of the selected option to be `bg-teal-dark`. What do you think? 
If it differs from the provided design then ignore this.

Reply:
yes looks nicer :+1: